src/tests/libpdbg_release_dt_root_test.c

```diff
@@ -0,0 +1,358 @@
+/* Copyright 2023 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <inttypes.h>
+#include <assert.h>
+
+#include <libpdbg.h>
+
```

Comment:
the test cases are not related to the functionality to switch back-ends it is more of adding additional test cases if so it should be in another branch and another PR and it should hav its own test file not part of release_dt_root_test.c

here we should have test cases only for releasing dt_root

Reply:
Corrected as suggested. Now the test cases test for the release and then switching of backend also